### PR TITLE
SISRP-9325 Add Capybara layer between Cucumber and Selenium

### PR DESCRIPTION
**Pull request by mDibyo**
https://jira.berkeley.edu/browse/SISRP-9325

### CODE REVIEW

**Comment by mDibyo** [comment]
Adding Capybara in between Cucumber and Selenium has decoupled Selenium webdriver from Cucumber. I am now using Poltergeist webdriver (uses headless PhantomJS as browser instead of firefox) instead of Selenium webdriver.
In a rather unscientific test, time taken for execution of tests (single trial) is down from 25 seconds to 12 seconds.

**Comment by mDibyo** [comment]
Ready for review.

File: test/features/step_definitions/website_navigation.rb [hunk 1]
```diff
@@ -1,27 +1,23 @@
-require 'selenium-webdriver'
 require 'json'
 
+# require '../support/hooks'
```
Comment (christianvuerings) [inline test/features/step_definitions/website_navigation.rb:3]: @mDibyo non-blocker but can we remove this?

Comment (mDibyo) [inline test/features/step_definitions/website_navigation.rb:3]: @christianvuerings Sure! I will remove it in the next PR.

**Pull request merged**